materialize-snowflake/snowflake.go

```diff
@@ -308,14 +308,10 @@ func (d *transactor) addBinding(target sql.Table) error {
 	b.store.mergeBounds = sql.NewMergeBoundsBuilder(target.Keys, d.ep.Dialect.Literal)
 
 	if b.target.DeltaUpdates && d.cfg.Credentials.AuthType == JWT {
-		var pipeName string
-		var err error
-		if pipeName, err = renderTableShardVersionTemplate(b.target, d._range.KeyBegin, d.version, d.templates.pipeName); err != nil {
-			return fmt.Errorf("pipeName template: %w", err)
-		} else {
-			pipeName = strings.ToUpper(strings.Trim(pipeName, "`"))
-			b.pipeName = fmt.Sprintf("%s.%s.%s", d.cfg.Database, d.cfg.Schema, pipeName)
-		}
+		var keyBegin = fmt.Sprintf("%08x", d._range.KeyBegin)
+		var tableName = strings.ReplaceAll(b.target.Path[len(b.target.Path)-1], " ", "_")
```

Comment:
I think this `tableName` part gets sanitized in `sanitizeAndAppendHash` already, so maybe the spaces don't need this replacement?

Reply:
You are right that is a leftover from before, removed 👍🏽